config/default.js

```diff
@@ -44,18 +45,58 @@ module.exports = {
         path: 'content',
         label: 'Content',
       },
+      {
+        path: 'users',
```

Comment:
Please follow the pattern in [content](https://github.com/scottnath/punchcard/blob/80084ed4e48f29117256ca3f819b1b23eaf4c2da/config/default.js#L17-L20)